src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -175,9 +175,6 @@ export async function makePythonDebugConfig(
         }
     }
     config.codeRoot = pathutil.normalize(config.codeRoot)
-    if (config.invokeTarget.target === 'template') {
-        config.codeRoot = path.dirname(config.codeRoot)
```

Comment:
what's the difference between `path.dirname()` and `pathutil.normalize()`

Reply:
They're unrelated.

- dirname() gets the parent directory of its argument.
- `pathutil.normalize()` is a function that tries to eliminate non-essential differences in a path string: https://github.com/aws/aws-toolkit-vscode/blob/c115cb61525cf986b620757ab07f134e288cbfbe/src/shared/utilities/pathUtils.ts#L60-L79